assets/scss/components/_navbar.scss

```diff
@@ -104,11 +104,12 @@ li {
 
 #item1,
 #item2,
-#item3 {
+#item3,
+#item4 {
```

Comment:
you could add this style to the item class

Reply:
actually I believe this is a duplicate of what you do on line 141

Reply:
ah yea, I see it

Reply:
should be cleaner now